pandas/tests/resample/test_resample_api.py

```diff
@@ -611,3 +611,76 @@ def test_resample_agg_readonly():
 
     result = rs.agg("min")
     tm.assert_series_equal(result, expected)
+
```

Comment:
is it possible to parametrize these tests?

Reply:
done